app/src/main/shared/reducers/authReducer.ts

```diff
@@ -18,7 +19,8 @@ export const initialAuthState: AuthState = {
     bearerToken: null,
     permissions: [],
     isAccountActive: false,
-    isModeller: false
+    isModeller: false,
+    canUploadCoverage: false
```

Comment:
Feel like a bit of a code-smell that we're providing both `permissions` and `modellingGroups` *and* derived fields such as `isAccountActive`, `isModeller` and `canUploadCoverage`. I worry that e.g. changing the definition of a `isModeller` could be difficult if code elsewhere is inferring it from `modellingGroups`. But I don't think it's specific to this PR and I'm not fully aware of broader context!

Reply:
Agreed. I want to centralise the isTypeOfUser and canDoSomething properties into the auth state as part of the work to lock down the other pages. Then the components which are currently working that stuff out themselves can use those properties in the auth state, and the router can use them too. We should be able to take the permissions list out of the state as a property at that point. I may try tackling this for our Weds afternoon maintenance session this week...